From: Hannes Reinecke <hare@suse.de>
Date: Tue, 12 Jun 2012 15:41:05 +0200
Subject: [PATCH] mach-dove: Fixup ge00 initialisation
Patch-Mainline: Submitted to linux-arm-kernel

The last argument of orion_ge00_init() is actually the error
interrupt, so we should be using the correct value here.

Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 arch/arm/mach-dove/common.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm/mach-dove/common.c
+++ b/arch/arm/mach-dove/common.c
@@ -102,7 +102,8 @@ void __init dove_ehci1_init(void)
 void __init dove_ge00_init(struct mv643xx_eth_platform_data *eth_data)
 {
 	orion_ge00_init(eth_data,
-			DOVE_GE00_PHYS_BASE, IRQ_DOVE_GE00_SUM, 0);
+			DOVE_GE00_PHYS_BASE, IRQ_DOVE_GE00_SUM,
+			IRQ_DOVE_GE00_ERR);
 }
 
 /*****************************************************************************
